docs/source/usage.ipynb

```diff
@@ -23,7 +24,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
We don't want to store the execution result in notebook. That way the nbsphinx won't actually run the notebook.
Please clear all output and then save again.